transform/transformer_test.go

```diff
@@ -467,7 +487,7 @@ func TestSaveValue(t *testing.T) {
 }
 
 func TestTransformer(t *testing.T) {
-	const parallelRuns = 4
+	const parallelRuns = 1
```

Comment:
One of the points of this test is to make sure multiple transforms can run from the same transformer concurrently, so changing this to `1` defeats the purpose of the test.

Reply:
ah yes. missed this in the post-clean up. will change back.